### PR TITLE
Safari Browser popup video not playing

### LINKED ISSUE #2553: Video - Safari Browser popup video not playing
**Describe the bug**
When you open a video in popup and symlink is enabled then the video is not playing in safari.

**To Reproduce**
1. Go to Activity OR Video pages in a group, user profile, and directory page
2. Open a video in the popup it's not playing.

**Expected behavior**
VIdeo Should play in popup in safari to

**Screenshots**

**Environment**
Which site did you replicate this issue on?

**Support ticket links**
If applicable, add HelpScout link or ticket number where the issue was originally reported.

### PR DESCRIPTION
**Pull request by krunal-buddyboss**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2553 

### How to test the changes in this Pull Request:

1. Go to Activity OR Video pages in a group, user profile, and directory page
2. Open a video in the popup it's not playing.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

**Comment by chetansatasiya** [comment]
@krunal-buddyboss Issue is still their in Safari if the symlink is disabled

File: src/bp-document/bp-document-functions.php [hunk 1]
```diff
@@ -4522,6 +4522,8 @@ function bb_document_video_get_symlink( $document, $generate = true ) {
 
 							if ( $generate ) {
 								// Generate Document Video Symlink.
+								$ext = pathinfo($attached_file, PATHINFO_EXTENSION);
+								$attachment_path .= ( ! empty( $ext ) ) ? '.' . $ext : '';
```
Comment (chetansatasiya) [inline src/bp-document/bp-document-functions.php:4526]: FIX PHPCS

File: src/bp-video/bp-video-functions.php [hunk 2]
```diff
@@ -3819,7 +3819,8 @@ function bb_video_get_symlink( $video, $generate = true ) {
 			$privacy             = $video->privacy;
 			$upload_directory    = wp_get_upload_dir();
 			$time                = time();
-			$attachment_path     = $video_symlinks_path . '/' . md5( $video->id . $attachment_id . $privacy . $time );
+			$ext = pathinfo($attached_file, PATHINFO_EXTENSION);
+			$attachment_path     = $video_symlinks_path . '/' . md5( $video->id . $attachment_id . $privacy . $time ) . ( ! empty( $ext ) ? '.' . $ext : '' );
```
Comment (chetansatasiya) [inline src/bp-video/bp-video-functions.php:3823]: FIX PHPCS

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
FIX PHPCS issues

**Review by chetansatasiya — APPROVED** [review]

**Comment by chetansatasiya** [comment]
@krunal-buddyboss Merge in dev and put in testing.

@mohit-buddyboss @sumitbuddyboss Make sure to test this in all the devices & servers

**Comment by krunal-buddyboss** [comment]
Thanks @chetansatasiya , I have merged in dev.

@mohit-buddyboss @sumitbuddyboss  Please test it. Thanks.

**Comment by sumitbuddyboss** [comment]
Hello @krunal-buddyboss 

I have checked the Safari browser issue in the Mobile devices and Different Servers and it's working fine for proof please check the below videos.

**Local server, Flywheel, Site ground, and Windows Server**
https://www.loom.com/share/794fb309c15d45f7b26721c01860b40d
https://www.loom.com/share/144a08326a274c75bdb49150c950d5dd 

**ios Mobile device**
https://www.loom.com/share/129873a5daec49969ed528d53f7b23f0

@sourabhmatolia @chetansatasiya 
Ready for release 🚀 

Thank you

**Pull request merged**